pages/community/precice-workshop-organizing.md

```diff
@@ -0,0 +1,120 @@
+
+### Find a venue
+
+If not at the same time as the date announcement, decide on a venue, and book rooms as soon as possible after that. Depending on the local workflows and availability, room booking requests might need to be done more than six months ahead, so aim to book at least the main room at the time that the venue is decided.
+
+Typical room requirements:
+
+- Fitting around 50 people (but not much larger than that)
+- For the hands-on training course and the user support sessions: easy access at every seat, (collaborative) working surface, enough power sockets, stable internet connection
+- For the talks: good acoustics and visibility, recording infrastructure
+- For the world café: (standing) tables, placed far from each other, isolated from noisy areas of the building
+- For the catering: ideally next to the rooms, isolated, standing
+
+With "recording infrastructure", we mean infrastructure to capture the speaker audio+video, slides, and an additional microphone in separate stream (a meeting camera system is typically not fitting the purpose). We can provide specialized hardware to do the mixing of these channels into what you later see as recordings on YouTube, if this is not already available in the room.
+
+The rooms for the training and the talks are typically different, even if the schedule is typically structured as a single track. The catering and the world café can typically use the same room and tables, but reusing the collaborative working room for the world café is also fine.
+
+### Define financing scheme
+
+Typically, the workshop is co-funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or an external company. The setup should allow to model an income to reinvest into maintaining preCICE.
+
+Multiple types of tickets are typically defined:
+
+- early -vs- late registration
+- full price -vs- discount for academia
+- training course included -vs- only main part of the workshop
+
+In addition, free registration is accounted for:
+
+- holders of a support license
+- invited speakers
+- organizers (if possible)
+- special cases
+
+### Set milestones
+
+Set clear responsible persons for each task, and track progress in a collaborative project board.
+
+Important dates to set:
+
+- "save the date" announcement: one year ahead
+- call for contributions (needs a form): open six months ahead. An early call mainly acts as advertisement and helps participants that need a visa application to travel.
+- open registration (needs a form): the earlier the better, and useful as a concrete action when advertising.
+- abstract submission deadline and contribution acceptance: the earlier the better, typically before the early-bird registration deadline. The contribution acceptance can be as short as one week after the deadline.
```

Comment:
maybe it would be good to add a date estimation here. I think 3-4 months before workshop is reasonable (otherwise might be hard to apply for a visa)